src/modules/PublicLab.RichTextModule.js

```diff
@@ -150,21 +150,30 @@ module.exports = PublicLab.RichTextModule = PublicLab.Module.extend({
 
     crossvent.add(_module.wysiwyg.editable, 'keydown', function (e) {
       _editor.validate();
-      if (_module.wysiwyg.mode == "wysiwyg" && _module.value().match(/\\\]\(|\\##|\\\*\\\*/g) && $('.markdown-warning').length === 0) {
+      var regexp = /\\\]\(|\\##|\\\*\\\*/g;
+      if (_module.wysiwyg.mode == "wysiwyg" && _module.value().match(regexp) && $('.markdown-warning').length === 0) {
         var message = "Looks like you're using <a href='http://wikipedia.org/en/Markdown'>Markdown</a> while in Rich Text mode. If you'd like to continue in Markdown mode, <a class='alert-change-mode' href='javascript:void();'>click here</a>.";
-        $(_module.wysiwyg.editable).after("<div class='markdown-warning alert alert-warning'>" + message + "</div>");
+        $(_module.wysiwyg.editable).after("<div id='scrollpointMD' class='markdown-warning alert alert-warning'>" + message + "</div>");
         $('.alert-change-mode').click(function alertChangeMode() {
           _module.setMode('markdown');
         });
+        $('html, body').animate({
+          scrollTop: $("#scrollpointMD").offset().top
```

Comment:
Could we make this `$(_module.wysiwyg.editable).find('#scrollpointMD')` so that it's relative to the current editor HTML, in case 2 editors are open in the same window?

Reply:
There's a problem with that it seems, as the element with id `#scrollpointMD` will always be outside of `$(_module.wysiwyg.editable)` area, i.e.,` $(_module.wysiwyg.editable).after("<div id='scrollpointMD' class='markdown-warning alert alert-warning'>" + message + "</div>");`